### ISSUE TITLE
Better integrate CMake with the Android build, instead of using the autotools version

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#499]
```
another test email
```

Original issue reported on code.google.com by `feedback...@gmail.com` on 8 Dec 2012 at 2:21
* Blocked on: #352

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
It's nice to have a single way to describe a project.
```

Original comment by `vittorio...@gmail.com` on 25 Mar 2013 at 1:27
* Changed title: **Better integrate CMake with the Android build, instead of using the autotools version**
* Changed state: **New**
* Added labels: **Priority-Low**, **OpSys-Android**, **Type-Enhancement**
* Now blocked on: #352
